qa/tasks/vstart_runner.py

```diff
@@ -1016,12 +1016,6 @@ def __init__(self, ctx):
     def admin_remote(self):
         return LocalRemote()
 
-    def set_config_opt(self, section, opt, val):
```

Comment:
ack. so we are moving this functionality from `CephCluster` side into `CephTestCase`.